frontend/src/components/FAQ/FAQNavBar.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react';
+import icon from '../images/home-icon.png'
+import { Link } from 'react-router-dom'
+import { Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
+
+const FAQNavBar = (): ReactElement => {
+  const [toggleMenu, setToggle] = React.useState(false)
+  const clickToggle = (): void => {
+    setToggle(!toggleMenu)
+  }
+  const [width, setWidth] = React.useState(0)
+  const update = (): void => {
+    setWidth(window.innerWidth)
+    if (width > 600) {
+      setToggle(false)
+    }
+  }
+
+  window.addEventListener('resize', update)
+  React.useEffect(() => {
```

Comment:
nit: instead of doing `React.useEffect` you can import `{ useEffect }` when importing React. Same goes for `useState`.